doc/nrf/libraries/networking/azure_iot_hub.rst

```diff
@@ -17,6 +17,11 @@ When the device provisioning is complete, the library automatically connects to
 The library also has integrated support for Azure FOTA.
 For more information on Azure FOTA, see the documentation on :ref:`lib_azure_fota` library and :ref:`azure_fota_sample` sample.
 
+.. warning::
```

Comment:
```suggestion
.. note::
```
A warning is only used when an operating procedure or practice, which, if not correctly followed, could result in personal injury or loss of life.

Reply:
Really? I checked the use in NCS, Zerphyr and west, and that doesn't seem to be the convention.
I'll change this to note, but it would be nice if we had a label for particularly important items between a note that doesn't really stand out from the crowd and a warning about serious bodily harm.

Reply:
I was made aware of the `.. important:` alternative. Is that an option here?

Reply:
Important is ok. Warning is not.